modules/mobile/spec/request/appointments_create_request_spec.rb

```diff
@@ -52,7 +52,8 @@
     it 'clears the cache' do
       expect(Mobile::V0::Appointment).to receive(:clear_cache).once
 
-      VCR.use_cassette('appointments/post_appointments_va_proposed_clinic_200', match_requests_on: %i[method uri]) do
+      VCR.use_cassette('appointments/post_appointments_va_proposed_clinic_200',
```

Comment:
I would guess these changes are leftover from the path change you made. Might be worth changing them back just to avoid having a bunch of unnecessary changes in the PR.

Reply:
It's actually rubocop. It won't pass without adding that newline so I think I'm out of luck

Reply:
Interestingly though it only forces the newline on some files. It let me revert them on this file for instance but kept adding it back for others

Reply:
Ok, that's cool. Just do whatever rubocop says.